src/styles/App.css

```diff
@@ -0,0 +1,329 @@
+@import url("https://fonts.googleapis.com/css?family=Dancing+Script");
+body {
```

Comment:
Good job @Oluwafayokemi. This file is generated (App.css), we should avoid committing generated files.